### PR TITLE
[Task #68279] Create SalonControllers and Route

### PR DESCRIPTION
**Pull request by Hungnd-0370**
## Related Tickets:
- [#68279](https://edu-redmine.sun-asterisk.vn/issues/68279)

## WHAT this PR does?
- Tạo file SalonController và code method index()
- tạo route cho salon
- tạo file giao diện test cho index 

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Review by tungbd-0214 — APPROVED** [review]

**Comment by Hungnd-0370** [comment]
ready

File: resources/js/Pages/salons/Salons.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Authenticated from '@/Layouts/Authenticated';
+import { Head } from '@inertiajs/inertia-react';
+
+export default function Dashboard(props) {
+    return (
+        <Authenticated
+            auth={props.auth}
+            errors={props.errors}
+            header={<h2 className="font-semibold text-xl text-gray-800 leading-tight">Dashboard</h2>}
```
Comment (ghost) [inline resources/js/Pages/salons/Salons.jsx]: localizatio

File: resources/js/Pages/salons/Salons.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        >
+            <Head title="Dashboard" />
+
+            <div className="py-12">
+                <div className="max-w-7xl mx-auto sm:px-6 lg:px-8">
+                    <div className="bg-white overflow-hidden shadow-sm sm:rounded-lg">
+                        <div className="p-6 bg-white border-b border-gray-200">List Salons</div>
```
Comment (ghost) [inline resources/js/Pages/salons/Salons.jsx]: localization

File: routes/web.php [hunk 3]
```diff
@@ -37,4 +38,12 @@ function () {
     }
 )->middleware(['auth', 'verified'])->name('dashboard');
 
+Route::get('/salons', [SalonController::class, 'index'])->name('salons.index');
+Route::get('/salons/create', [SalonController::class, 'create'])->name('salons.create');
+Route::get('/salons/{salon}/edit', [SalonController::class, 'edit'])->name('salons.edit');
+Route::get('/salons/{salon}', [SalonController::class, 'show'])->name('salons.show');
+Route::post('/salons', [SalonController::class, 'store'])->name('salons.store');
+Route::put('/salons/{salon}', [SalonController::class, 'update'])->name('salons.update');
+Route::delete('/salons/{salon}', [SalonController::class, 'destroy'])->name('salons.destroy');
```
Comment (ghost) [inline routes/web.php]: gom vao thanh route resource

**Comment by Hungnd-0370** [comment]
em xin phép tạo view sang task riêng ạ

**Comment by Hungnd-0370** [comment]
ready

**Pull request merged**